cmd/thanos/rule.go

```diff
@@ -200,6 +217,35 @@ func runRule(
 			close(cancel)
 		})
 	}
+	{
+		l, err := net.Listen("tcp", grpcAddr)
+		if err != nil {
+			return errors.Wrap(err, "listen API address")
+		}
+		logger := log.With(logger, "component", "store")
+
+		store := store.NewTSDBStore(logger, reg, db, lset)
+
+		met := grpc_prometheus.NewServerMetrics()
+		met.EnableHandlingTimeHistogram(
+			grpc_prometheus.WithHistogramBuckets([]float64{
+				0.001, 0.01, 0.05, 0.1, 0.2, 0.4, 0.8, 1.6, 3.2, 6.4,
```

Comment:
Can we ensure that all of these are the same across all thanos Grpc clients/servers i.e fetch it from common var

Reply:
Do we want that though? Right now they are the same but they are also just guesses at this point. 
We might find out different components have different performance characteristics  and different buckets could make sense.

Reply:
sgtm